script-test/playbackstrategies/msestrategytest.js

```diff
@@ -1,491 +1,579 @@
+          mockDashInstance = jasmine.createSpyObj('mockDashInstance',
+            ['initialize', 'getDebug', 'getSource', 'on', 'off', 'time', 'duration', 'attachSource',
+              'reset', 'isPaused', 'pause', 'play', 'seek', 'isReady', 'refreshManifest', 'getDashMetrics', 'getMetricsFor']);
 
-        mockDashInstance.duration.and.returnValue(101);
+          mockDashInstance.duration.and.returnValue(101);
 
-        mockDashjs.MediaPlayer.and.returnValue(mockDashMediaPlayer);
-        mockDashMediaPlayer.create.and.returnValue(mockDashInstance);
+          mockDashjs.MediaPlayer.and.returnValue(mockDashMediaPlayer);
+          mockDashMediaPlayer.create.and.returnValue(mockDashInstance);
 
-        mockVideoElement.addEventListener.and.callFake(function (eventType, handler) {
-          eventHandlers[eventType] = handler;
+          mockVideoElement.addEventListener.and.callFake(function (eventType, handler) {
+            eventHandlers[eventType] = handler;
 
-          eventCallbacks = function (event) {
-            eventHandlers[event].call(event);
-          };
-        });
+            eventCallbacks = function (event) {
+              eventHandlers[event].call(event);
+            };
+          });
+
+          // For DVRInfo Based Seekable Range
+          mockDashInstance.isReady.and.returnValue(true);
+          mockDashInstance.getDashMetrics.and.returnValue({
+            getCurrentDVRInfo: function () {
+              return {range: {
+                start: 0,
+                end: 101
+              }};
+            }
+          });
+
+          window.dashjs = mockDashjs;
 
-        // For DVRInfo Based Seekable Range
-        mockDashInstance.isReady.and.returnValue(true);
-        mockDashInstance.getDashMetrics.and.returnValue({
-          getCurrentDVRInfo: function () {
-            return {range: {
-              start: 0,
-              end: 101
-            }};
-          }
+          spyOn(document, 'createElement').and.returnValue(mockVideoElement);
+          mockDashInstance.getDebug.and.returnValue(mockDashDebug);
+        }
+
+        afterEach(function () {
+          mockVideoElement.currentTime = 0;
+          document.body.removeChild(videoContainer);
         });
 
-        window.dashjs = mockDashjs;
+        describe('Transitions', function () {
+          it('canBePaused() Transition is true', function () {
+            setUpMSE();
 
-        spyOn(document, 'createElement').and.returnValue(mockVideoElement);
-        mockDashInstance.getDebug.and.returnValue(mockDashDebug);
-      }
+            expect(mseStrategy.transitions.canBePaused()).toBe(true);
+          });
 
-      afterEach(function () {
-        mockVideoElement.currentTime = 0;
-        document.body.removeChild(videoContainer);
-      });
+          it('should create an audio element and add it to the video container', function () {
```

Comment:
Can we rename `videoContainer`? It's called `playbackElement` in the implementation, and is no longer just for video as of this change.